spack/packages/sirius/package.py

```diff
@@ -263,18 +241,16 @@ def cmake_args(self):
             args.append(self.define(cm_label + "ELPA_INCLUDE_DIR", elpa_incdir))
 
         if "+cuda" in spec:
-            cuda_arch = self.spec.variants["cuda_arch"].value
+            cuda_arch = spec.variants["cuda_arch"].value
             if cuda_arch[0] != "none":
-                args += [self.define("CMAKE_CUDA_ARCHITECTURES", cuda_arch)]
+                # Make SIRIUS handle it
+                if "@6:7.4.3" in spec:
```

Comment:
didn't we retire <7 versions of sirius?

Reply:
Yes, the lower bound is actually not neede here, I made it `:7.4.3` now.